### ISSUE TITLE
A more legible dice font

### ISSUE THREAD

**Issue by calculuschild** [naturalcrit/homebrewery#3465]
Recording this here since I think the idea has merit.

The DiceFont we have now is serviceable, but @Gazook89 presented what I think is a really nice alternative that allows larger characters, and is thus more legible to my eye.

> I would rather keep a consistent dice shape, and add a "stroke" around the numbers. The stroke wouldn't be another color, but just an 'erased' stroke...and the actual svg wouldn't actually have a true stroke.

<img alt="image" width="491" src="https://private-user-images.githubusercontent.com/58999374/329122646-72d5750b-c31a-4175-ac22-fd2e9d4b9a29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyNjM5OTEsIm5iZiI6MTcxNTI2MzY5MSwicGF0aCI6Ii81ODk5OTM3NC8zMjkxMjI2NDYtNzJkNTc1MGItYzMxYS00MTc1LWFjMjItZmQyZTlkNGI5YTI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA5VDE0MDgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwODM4MzZlZmRmZGQ2MTQxNjM3MzdiZGE3OTBmM2ViODhlNDFhOTQxOWQxZTA2MDkwZGQ3MWUyY2RlZmNmYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pcqaLO6FItavG2zAO2ZUFuO7GDUCidxyH6QNstYZQ0Q">

If anyone wants to make a "diceFont Redux" as a side project.

**Comment by calculuschild** [comment]
And for fun, here's a crack at a D100.

![image](https://github.com/naturalcrit/homebrewery/assets/5878534/b0172b1e-10cf-487c-a4a5-fcde1734a06c)

**Comment by 5e-Cleric** [comment]
Definitely prefer the second option, way more space between numbers and other lines

**Comment by Gazook89** [comment]
![d10-10-big](https://github.com/naturalcrit/homebrewery/assets/58999374/efe8e32b-e213-4402-9fc3-abe2d3e87b9f)

Here is a better take on the 'stroked' numbers.  Affinity Designer has a "Contour Tool" that can expand the edges of a shape, like a stroke, but keep the edge of that stroke as a vector path.   I'm sure InDesign and Inkscape have similar (I know inkscape has stroke-to-path capabilities which is similar).  

So my process here was:

1. Open existing glyph svg in Designer
2. Duplicate the number layer (an SVG curve)
3. Use Contour Tool on the duplicated layer, and set radius to .6px
4. Select the dice shape layer, and the duplicated layer, and use the Subtract operation
5. export to SVG.

Unfortunately, Designer doesn't have any scripting. Other programs do.

**Comment by Gazook89** [comment]
I had a thought, more of just a "maybe this would be cool to try":  A variable font that used multiple alternate glyphs on a single axis.....applied here, it would be a single character that would change the displayed glyph based on something like the `font-weight`.   So a `d10` character, that when set to `font-weight: 1` would display a d10 glyph with the number 1 showing.   Change the font-weight to 2, and it would be the same character but display a 2 on it.

This isn't useful at all, I don't think, but you can read more about glyph swapping here.  https://glyphsapp.com/learn/alternating-glyph-shapes

**Comment by 5e-Cleric** [comment]
> This isn't useful at all, I don't think, but you can read more about glyph swapping here. https://glyphsapp.com/learn/alternating-glyph-shapes

I disagree, variable fonts are great! i just don't know exactly how they work just yet, or how to make them! If i did we would likely be using them.

**Comment by 5e-Cleric** [comment]
Not sure its worth our time tbh, considering closing.
